apps/src/feedback.js

```diff
@@ -898,16 +898,18 @@ FeedbackUtils.prototype.showGeneratedCode = function(Dialog) {
 
 /**
  * Display the "Clear Puzzle" confirmation dialog.  Calls `callback` if the user
- * confirms they want to clear the puzzle.
+ * confirms they want to clear the puzzle.  Takes optional parameter to hide the
+ * icon.
  */
-FeedbackUtils.prototype.showClearPuzzleConfirmation = function(Dialog, callback) {
+FeedbackUtils.prototype.showClearPuzzleConfirmation = function(Dialog, callback, hideIcon) {
```

Comment:
I think convention would be to put the `callback` argument last here.  There's only the one caller to this method, so it should be a pretty painless change.